ta/pkcs11/src/attributes.h

```diff
@@ -215,12 +215,18 @@ static inline enum pkcs11_rc get_u32_attribute(struct obj_attrs *head,
 /*
  * Return true if all attributes from the reference are found and match value
  * in the candidate attribute list.
- *
- * Return PKCS11_CKR_OK on success, or a PKCS11 return code.
  */
 bool attributes_match_reference(struct obj_attrs *ref,
 				struct obj_attrs *candidate);
 
+/*
+ * Check if attributes from @ref are all found in @head and if not are these
```

Comment:
"... and if not **~~are~~* these are added..."
Shorter other suggestion:
` * Check attributes from @ref are all found or added into @head`

Reply:
Changed to " Check attributes from @ref are all found or added into @head"